app/models/chf/citable_attributes.rb

```diff
@@ -475,5 +482,48 @@ def archive_location
       end
     end
 
+    class TreatAsOralHistory < StandardTreatment
+
+      def title
+        "#{parse_name(work.interviewee.first)}, interviewed by #{parse_name(work.interviewer.first)} at #{work.place_of_interview.first} on #{work.date_of_work.first}"
+      end
+
+      def csl_type
+        "interview"
+      end
+
+      def authors
+        [] # Having no "authors" is regrettable, but such is the consequence
+        # of having ruby-csl treat this as an "interview".
+      end
+
+      def medium
+        nil
+      end
+
+      def publisher_place
+        'Philadelphia'
+      end
+
+      def publisher
+        'Science History Institute'.freeze
+      end
+
+      def date
+        return nil if work.date_of_work.length == 0
+        begin
+          date_recorded = Date.strptime(work.date_of_work.first, '%Y-%m-%d')
+        rescue ArgumentError
+          return nil
+        end
+        date_recorded ? CiteProc::Date.new([date_recorded.year]) : nil
```

Comment:
i think we'd rather the whole date with month and day for these, not just the year. To more closely match suggested format from catalog. 

I'm not totally sure how you create a CiteProc::Date like that either, might have to look at CiteProc code in that gem.